rego/rego_wasmtarget_test.go

```diff
@@ -56,7 +56,7 @@ func TestPrepareAndEvalWithWasmTarget(t *testing.T) {
 		t.Fatalf("Unexpected error: %s", err)
 	}
 
-	assertPreparedEvalQueryEval(t, pq, []EvalOption{}, "[[true, true]]")
+	assertPreparedEvalQueryEval(t, pq, []EvalOption{}, "[[true, true],[true, true]]")
```

Comment:
💭 not the most useful assertion, I suppose.

Reply:
Agreed that a result set with unique bindings would be more obvious but I'm fine either way.